test/application/onerror.js

```diff
@@ -9,9 +9,7 @@ describe('app.onerror(err)', function(){
   it('should throw an error if a non-error is given', function(done){
     const app = new Koa();
 
-    (function(){
-      app.onerror('foo');
-    }).should.throw(AssertionError, {message: 'non-error thrown: foo'});
+    (() => app.onerror('foo')).should.throw(AssertionError, {message: 'non-error thrown: foo'});
```

Comment:
I was trying to keep it under 80 chars, but there's a few longer lines so I just moved this to the same line after a comment.